src/components/select/select.yaml

```diff
@@ -17,6 +17,12 @@ examples:
         value: 3
         text: GOV.UK frontend option 3
         disabled: true
+      -
```

Comment:
Generally we're trying to only add to these examples when there's a visual difference that we need to be able to eyeball in the review app – we can omit this because we've got tests that cover this functionality.

It'd be good if we could remove this addition, and revert the corresponding changes to the README.

(sorry that this isn't documented anywhere – we must make this clearer)